src/functions/proof-of-reserve/proof-of-reserve-functions.ts

```diff
@@ -18,20 +19,22 @@ export async function verifyVaultDeposit(
   bitcoinBlockchainBlockHeight: number,
   bitcoinBlockchainAPI: string,
   bitcoinNetwork: Network
-): Promise<boolean> {
+): Promise<number> {
   try {
-    const fundingTransaction = await fetchBitcoinTransaction(
-      vault.fundingTxId,
-      bitcoinBlockchainAPI
-    );
+    if (!isStringDefinedAndNotEmpty(vault.wdTxId) && !isStringDefinedAndNotEmpty(vault.fundingTxId))
+      return 0;
```

Comment:
maybe you can come up with something shorter for this, as this function name is so long it's hard to read this string. 

```
function isValid(value) {
  return value !== undefined && value !== 0 && value !== '';
}
```

```
function isStrictlyValid(value) {
  return value !== undefined && value !== 0 && value !== '' && value !== null && value !== false && !isNaN(value);
}
```